<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Castle Engine Library Tester</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Screenshot"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuViewpoints">
    <property name="title">
     <string>Viewpoints</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpenGL_Information"/>
    <addaction name="actionShow_Warnings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSSAO"/>
    <addaction name="actionHead_Bobbing"/>
    <addaction name="actionMouse_Look"/>
    <addaction name="actionMultiSampling"/>
    <addaction name="actionHeadlight"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuViewpoints"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionWalk"/>
   <addaction name="actionFly"/>
   <addaction name="actionExamine"/>
   <addaction name="actionTurntable"/>
   <addaction name="separator"/>
   <addaction name="actionPrevView"/>
   <addaction name="actionNextView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionWalk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Walk</string>
   </property>
  </action>
  <action name="actionFly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fly</string>
   </property>
  </action>
  <action name="actionExamine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Examine</string>
   </property>
  </action>
  <action name="actionTurntable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Turntable</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionNextView">
   <property name="text">
    <string>Next View</string>
   </property>
  </action>
  <action name="actionPrevView">
   <property name="text">
    <string>Prev View</string>
   </property>
  </action>
  <action name="actionOpenGL_Information">
   <property name="text">
    <string>OpenGL Information</string>
   </property>
  </action>
  <action name="actionShow_Warnings">
   <property name="text">
    <string>Show Warnings</string>
   </property>
  </action>
  <action name="actionSSAO">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SSAO</string>
   </property>
  </action>
  <action name="actionHead_Bobbing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Head Bobbing</string>
   </property>
  </action>
  <action name="actionMouse_Look">
   <property name="text">
    <string>Mouse Look</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionMultiSampling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MultiSampling</string>
   </property>
  </action>
  <action name="actionHeadlight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Headlight</string>
   </property>
  </action>
  <action name="actionSave_Screenshot">
   <property name="text">
    <string>Save Screenshot...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
